airbyte-integrations/connectors/source-google-ads/source_google_ads/google_ads.py

```diff
@@ -214,8 +214,12 @@ def get_field_value(field_value: GoogleAdsRow, field: str, schema_type: Mapping[
         return field_value
 
     @staticmethod
-    def parse_single_result(schema: Mapping[str, Any], result: GoogleAdsRow):
+    def parse_single_result(schema: Mapping[str, Any], result: GoogleAdsRow, nullable: Optional[List[str]] = None):
+        if nullable is None:
+            nullable = []
         props = schema.get("properties")
         fields = GoogleAds.get_fields_from_schema(schema)
-        single_record = {field: GoogleAds.get_field_value(result, field, props.get(field)) for field in fields}
+        single_record = {
+            field: GoogleAds.get_field_value(result, field, props.get(field)) if field not in nullable else None for field in fields
```

Comment:
I have two questions with this line:
* My understanding of the issue is that we were passing fields that Google Ads would break on. 
* I don't see `get_field_value` using it's third argument `schema_type` so why do we need to change the caller? Shouldn't we just remove the argument from `get_field_value` instead? 

Based on these two questions, why is this change needed?

Reply:
After a sync discussion, it seems like the values for "user_interest.availabilities" and "user_interest.launched_to_all" have changed following the 500 errors and it is unexpected so until we understand the issue with the 500 errors, we will nullify those two fields so we will add a comment to reflect

Reply:
Made a comment with explanation what is going on